proprietary/tokens/src/common/ams/typography.tokens.json

```diff
@@ -0,0 +1,61 @@
+{
+  "ams": {
+    "common": {
```

Comment:
Do we really need this prefix? We don't previx component properties `ams-components-`?